packages/next/src/withTRPC.tsx

```diff
@@ -54,6 +54,7 @@ function transformQueryOrMutationCacheErrors<
 export type WithTRPCConfig<TRouter extends AnyRouter> =
   CreateTRPCClientOptions<TRouter> & {
     queryClientConfig?: QueryClientConfig;
+    abortOnUnmount?: boolean;
```

Comment:
not sure if we should add `OnUnmount` to the name. Technically, requests will be aborted when all observers have unsubscribed. In future versions of react, this might happen not only on unmounting, but also e.g. with a proposed new offscreen api when the observer goes offscreen but still remains mounted.

I would say that `abort?: boolean` would be fine, too.

Reply:
I'll leave is as-is for now, `abort` is a bit unclear to me what it means and we can always rename and deprecate later.

If we figure out a better name ahead of `10.0.0` we can rename it too.